client/packages/invoices/src/utils.ts

```diff
@@ -37,19 +37,22 @@ export const inboundStatuses: InvoiceNodeStatus[] = [
   InvoiceNodeStatus.Picked,
   InvoiceNodeStatus.Shipped,
   InvoiceNodeStatus.Delivered,
+  InvoiceNodeStatus.Received,
   InvoiceNodeStatus.Verified,
 ];
 
 export const manualInboundStatuses: InvoiceNodeStatus[] = [
   InvoiceNodeStatus.New,
   InvoiceNodeStatus.Delivered,
+  InvoiceNodeStatus.Received,
   InvoiceNodeStatus.Verified,
 ];
 
 export const nextStatusMap: { [k in InvoiceNodeStatus]?: InvoiceNodeStatus } = {
   [InvoiceNodeStatus.New]: InvoiceNodeStatus.Delivered,
   [InvoiceNodeStatus.Shipped]: InvoiceNodeStatus.Delivered,
-  [InvoiceNodeStatus.Delivered]: InvoiceNodeStatus.Verified,
+  [InvoiceNodeStatus.Delivered]: InvoiceNodeStatus.Received,
```

Comment:
It feels like the idea of all of those, is for them to be use in the StatusChange Toolbars, you are right about refactor